internal/static/traces/config.go

```diff
@@ -598,7 +597,7 @@ func resolver(config map[string]interface{}) (map[string]interface{}, error) {
 func (c *InstanceConfig) loadBalancingExporter() (map[string]interface{}, error) {
 	exporter, err := exporter(RemoteWriteConfig{
 		// Endpoint is omitted in OTel load balancing exporter
-		Endpoint:    "noop",
+		Endpoint:    "noop:8888",
```

Comment:
Why did this need to change? What is 8888?

Reply:
Tests were erroring - I think it's down to this extra validation check upstream:
```
  - `otelcol.exporter.otlp`: Checks for port in the config validation for the otlpexporter.
    https://github.com/open-telemetry/opentelemetry-collector/issues/9505
```
8888 is just a random port.

Reply:
Well, it's not too random. I chose 8888 because it is the default port on which the OTel collector reports metrics. Anyway, it doesn't matter because it's just a placeholder :)